CIME/hist_utils.py

```diff
@@ -72,6 +72,8 @@ def copy_histfiles(case, suffix, match_suffix=None):
     num_copied = 0
     for model in _iter_model_file_substrs(case):
         if case.get_value("TEST") and archive.exclude_testing(model):
+            logger.info("Case is a test and component %r is excluded from comparison")
```

Comment:
What does `%r` mean?

Reply:
@jgfouca Fixed, forgot to add the `model` argument.